src/Sena/M9_Interface/Summary.java

```diff
@@ -0,0 +1,52 @@
+package Sena.M9_Interface;
+
+public class Summary {
+
+    /**
+     * Presentation link:
+     * https://docs.google.com/presentation/d/18Vl4zrmA-M5PoJX65bBD-VvQ3aDyWqmL_ItfK1BNfwc/edit?usp=sharing
+     */
+
+    // Interface
+    // In the Java programming language, an interface is a reference type, similar to a class,
+    // that can contain only constants, method signatures, default methods, static methods, and nested types.
+    // Interfaces are 100% abstract
```

Comment:
yesterday I  had doubt about 100% abstraction, because we can have default methods.. right? so is it still 100% abstraction, thats what I was thinking about

Reply:
Thank you so much! I'll add that "default method" is an exception, but are they really important for them to keep in mind? I'll not go over it in detail accordingly...